inc/customizer/controls/react/src/icon-picker/IconFieldComponent.js

```diff
@@ -0,0 +1,12 @@
+import IconField from './IconField.js';
```

Comment:
Same thing here. As the IconField is not a control itself, it has no wrapper component that will interact with the customize API.